code-challenges.test.js

```diff
@@ -16,133 +16,163 @@
-// })
-// })
-
-// var randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// // // // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-// // // // var randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
-// // // // // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
-
-// // // // b) Create the function that makes the test pass.
-// // // Psuedo coding:
-// // // we are going to need to access the string in our array 
-// // // we need to split the strings so we can gain access to index 0 of each string in each array.
-// // // we need to use the built in function .toUpperCase on each of our srings on the 0 index. only
-// // // then we need to join the array back together. 
-// // Psuedo Coding:
-// //first we creat a funtion called "stringCapitalizer"
-// //then we'll use the built in method .map to access the zeroeth value of each string in our array. 
-// // then we will return the string back together 
-// // const stringCapitalizer = (array) => {
-// // let allNouns = array.map(value => {
-// //     return value[0].toUpperCase() + value.substring(1)
-// // })
-// // return allNouns.join([" "])
-// // }
-// // console.log(stringCapitalizer(randomNouns1));
+ // a) Create a test with expect statements for each of the variables provided.
+Psuedo coding:
+Our test id going to run a function called "stringCapitalizer"
+It is going to take our array of strings and return it with the zeroeth index of each string capitalized, then join the string back together.
+
+describe("stringCapitalizer", () => {
+test("returns an array with all the strings at their 0 index capitalized", () => {
+expect(stringCapitalizer(randomNouns1)).toEqual(["Streetlamp Potato Teeth Conclusion Nephew"])
+})
+})
+
+var randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+// // // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+var randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
+// // // // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
+
+// b) Create the function that makes the test pass.
+Psuedo coding:
+we are going to need to access the string in our array 
+we need to split the strings so we can gain access to index 0 of each string in each array.
+we need to use the built in function .toUpperCase on each of our srings on the 0 index. only
+ then we need to join the array back together. 
+// Psuedo Coding:
+first we creat a funtion called "stringCapitalizer"
+then we'll use the built in method .map to access the zeroeth value of each string in our array. 
+then we will return the string back together 
+
+const stringCapitalizer = (array) => {
+let allNouns = array.map(value => {
+    return value[0].toUpperCase() + value.substring(1)
+})
+return allNouns.join([""])
+}
+console.log(stringCapitalizer(randomNouns1));
```

Comment:
The logic here is pretty good! I think calling your variable all nouns is pretty confusing when it's capitalizing. the first letter.